script/collect.js

```diff
@@ -118,11 +118,14 @@ async function main () {
   const oldNpmCount = old.find(release => release.npm_package_name === 'electron').length
   const newNpmCount = releases.find(release => release.npm_package_name === 'electron').length
 
+  const diffNotesCount = old.some((release, index) => release.body !== releases[index].body)
```

Comment:
This is no longer a Number. Now it's a Boolean. So it should be named something like `releaseBodyChanged`